mapproxy/util/escape.py

```diff
@@ -0,0 +1,10 @@
+def escape_html(data):
```

Comment:
as it also escapes JS (at least it states), another method name could be used. but this is not blocking

Reply:
I named it this way because it is used in html contexts ... More precise would be `escape_in_html`

Reply:
thanks for clarification